app/views/surveys/select_language.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Select Language</h1>
+
+<ul>
+  <% @languages.each do |language| %>
+    <li><%= link_to language, start_survey_survey_path(@survey, language: language) %></li>
```

Comment:
Language names should be written in the appropriate language. Spanish should be Español for example.

Reply:
yes, Needs to be changed in seeded values